src/test/resources/ProgramAB/bots/pikachu/aiml/pikachu.aiml

```diff
@@ -1,10 +1,8 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<aiml>
-	<category>
-		<pattern>私は*です</pattern>
-		<!--  あなたを*に会うのはいいことだ   ?? -->
-		<!--  あなたを*に会えてうれしい -->
-		<!-- あなたに会えてよかったXXXXX -->
-		<template>あなたに会えてよかった<star /></template>
-	</category>
-</aiml>
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
+<aiml>
+<!--  -->
```

Comment:
please revert this change.

Reply:
programab should no longer overwrite this file when the test runs..

Reply:
reverted to develop version